presets/inference/text-generation/requirements.txt

```diff
@@ -1,15 +1,17 @@
 # Dependencies for TFS
 
 # Core Dependencies
-transformers==4.41.2
-torch==2.2.0
+transformers>=4.45.0
+vllm==0.6.3
+torch==2.4.0
 accelerate==0.30.1
 fastapi>=0.111.0,<0.112.0  # Allow patch updates
-pydantic==2.7.4
+pydantic>=2.9
 uvicorn[standard]>=0.29.0,<0.30.0  # Allow patch updates
 peft==0.11.1
-numpy==1.22.4
+numpy
 sentencepiece==0.2.0
+jinja2>=3.1.0
```

Comment:
make separate requirements.txt file for vllm, lets not mess with something thats working. these pip packages have been pinned to very specific versions that are all compatible with each other

Reply:
Good point

Reply:
use different venv for each backend.